python/foglamp/services/core/api/service.py

```diff
@@ -40,6 +41,7 @@
     | GET POST            | /foglamp/service                                      |
     | GET                 | /foglamp/service/available                            |
     | GET                 | /foglamp/service/installed                            |
+    | PUT                 | /foglamp/service/{type}/{name}/update                 |
```

Comment:
this can be `/foglamp/service/{name}/update`, We can identify type if type based on process name, and check with a set of ['notification]`

Reply:
Ignore this!